terraform/deployments/govuk-eks/www_origin.tf

```diff
@@ -0,0 +1,76 @@
+# TODO: This is a placeholder service that will be replaced by `router` service.
+# The service will still be deployed later by using `kubectl` or `helm`.
+# The k8s ingress needs the service(s) to be present before creation or else
+# it will stuck in creation.
+# It is easier to have k8s ingress created in terraform since we can add a CNAME
+# to the "random" FQDN of the AWS ALB created to represent the k8s ingress.
```

Comment:
We won't want to use Terraform to manage k8s objects on the application side like this, otherwise we'll lose a lot of the testability and portability benefits of k8s. A much better way to make a CNAME to the ALB is to use [external-dns](https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/aws.md) and keep everything with k8s.

Reply:
cool, this adds to my knowledge. I only search whether the ingress controller could do it with annotations.